### PR TITLE
Update logger styling

### PR DESCRIPTION
**Pull request by MattElcock**
### What is the context of this PR?

The logs produced by Author and the API can be confusing to read when debugging as they aren't styled very well. I've introduced a prettier that'll only run locally with the hope that it'll make debugging easier.

I've excluded publisher from this as we're currently rebuilding it.

### How to review 

* Import the **Capability examples** questionnaire from pre-prod Author into your local environment and:
    * ensure it can be opened in Author;
    * then, ensure it can be viewed in Runner by pressing the **view survey** button
* Does this require a migration? We need one if existing JSON schema properties change
* Make the code looks OK
* Make sure the logs look OK

### What to do after everything is green

1. * [ ] Bring this branch up-to-date with Origin/Master
2. * [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. * [ ] Click the **Merge** button on this pull request
4. * [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. * [ ] Move the Trello card for this task into the next stage of the process

### CODE REVIEW

File: eq-author-api/server.js [hunk 1]
```diff
@@ -135,8 +131,8 @@ const createApp = () => {
   const httpServer = http.createServer(app);
   server.installSubscriptionHandlers(httpServer);
 
-  logger.info(`🚀 Server ready at ${server.graphqlPath}`);
-  logger.info(`🚀 Subscriptions ready at ${server.subscriptionsPath}`);
+  logger.info(`🚀  Server ready at ${server.graphqlPath}`);
+  logger.info(`🚀  Subscriptions ready at ${server.subscriptionsPath}`);
```
Comment (tjamesmac) [inline eq-author-api/server.js]: thank you

File: eq-author-api/utils/logger.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+const noir = require("pino-noir");
+
+let logger;
+
+switch (process.env.NODE_ENV) {
+  case "development": {
+    logger = require("pino")({
+      prettyPrint: {
+        colorize: true,
+        ignore: "pid,hostname",
```
Comment (tjamesmac) [inline eq-author-api/utils/logger.js]: I like that

**Comment by martyncolmer** [comment]
Screenshot 2020-03-03 at 17 02 47

File: eq-author-api/utils/logger.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    break;
+  }
+  default: {
+    logger = require("pino")();
+  }
+}
+
+const expressLogger = require("express-pino-logger")({
+  logger,
+  serializers: noir(["req.headers.authorization"], "[Redacted]"),
```
Comment (tjamesmac) [inline eq-author-api/utils/logger.js]: I think it might be something to do with this

**Pull request merged**